src/core/util/sys_vars.h

```diff
@@ -479,14 +479,12 @@ struct mce_sys_var {
     uint32_t tcp_send_buffer_size;
     uint32_t tx_segs_ring_batch_tcp;
     uint32_t tx_segs_pool_batch_tcp;
-    /* This field should be used to store and use data for XLIO_EXTRA_API_IOCTL */
+    /* This field should be used to store and use data for XLIO Socket API */
```

Comment:
Comment is misleading. The field name is self descriptive, so no need in such comment.